models/dex/dex_amm_trades.sql

```diff
@@ -0,0 +1,42 @@
+{{ config(
+        alias ='amm_trades',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "sector",
+                                "dex",
+                                \'["jeff-dude", "bh2smith"]\') }}'
```

Comment:
This file was actually renamed and the only "actual diff" is that alias has been changed to 'amm_trades' and I added myself to the contributors list (which isn't all that necessary).

Reply:
You should definitely add your name in there 👍

Reply:
ok then I will put it back...